internal/pkg/convert/convert.go

```diff
@@ -165,7 +166,10 @@ func getN9SLObjects(
 
 	// For each SLO object, create a Nobl9 SLO object.
 	for _, slo := range objects {
-		s := slo.(v1.SLO)
+		s, ok := slo.(v1.SLO)
+		if !ok {
+			return errors.New("for command convert only apiVersion 'openslo/v1' is supported")
```

Comment:
Can this be reworded? Maybe to "the convert command is only supported for apiVersion 'openslo/v1'"?